README.md

```diff
@@ -18,6 +18,20 @@ To start using Liberator add the following to your `build.sbt` file:
 libraryDependencies += "io.aecor" %% "liberator" % "0.1.0"
 ```
 
+Also you have to add macro related stuff: 
+
+```scala
+scalacOptions ++= Seq(
```

Comment:
This can be reduced to
```scala
scalacOptions += "-Ypartial-unification"
addCompilerPlugin("org.scalameta" % "paradise" % "3.0.0-beta4" cross CrossVersion.full)
```
It worth mentioning that `-Ypartial-unification` only available on TLS 2.11.8 for now, and that it will be available soon on LBS 2.11.9.
```scala
scalaOrganization := "org.typelevel"
```
Also I think it's better to merge these lines with first block on line 18, like this:
```scala
scalaOrganization := "org.typelevel"
libraryDependencies += "io.aecor" %% "liberator" % "0.1.0"
scalacOptions += "-Ypartial-unification"
addCompilerPlugin("org.scalameta" % "paradise" % "3.0.0-beta4" cross CrossVersion.full)
```